src/ucp/core/ucp_worker.c

```diff
@@ -887,6 +887,20 @@ static int ucp_worker_iface_support_keepalive(ucp_worker_iface_t *wiface)
                               UCT_IFACE_FLAG_EP_KEEPALIVE);
 }
 
+/* Compare target iface atomic capabilities to candidate's, and decide if candidate contains all target's capabilities */
```

Comment:
`candidate` doesn't exist in func sig anymore, so please adjust the comment text accordingly.